### DISCUSSION TITLE
Wrong structure?

### DISCUSSION THREAD

**Discussion by zelenskaya** [community/community#21687] [Other Feature Feedback, Questions, & Ideas]
<p>Hi! I created a gh-pages site at <a href="http://www.zelenskaya.github.io" rel="nofollow noopener">www.zelenskaya.github.io</a></p>
<p>I have set up a repository called kawunu and set up a new branch called gh-pages.</p>
<p>I have created index.html in that repository and committed.</p>
<p>What is the correct public URL?</p>
<p><a href="https://zelenskaya.github.io/kawunu/index.html" rel="nofollow noopener">https://zelenskaya.github.io/kawunu/index.html</a> is not working.</p>
<p>how can i make <a href="https://zelenskaya.github.io/kawunu/index.html" rel="nofollow noopener">https://zelenskaya.github.io </a>work?</p>
<p>Thanks!</p>
<p>Anna</p>

**Comment by carlogilmar** [comment, marked as answer]
<p>I know that if you enable GitHub Pages over your repository, your url will be: your_github_username.github.io/your_repository_name</p>
<p>You can see this in your case: <a href="https://zelenskaya.github.io/experiments/" rel="nofollow noopener">https://</a><a href="https://zelenskaya.github.io/experiments/" rel="nofollow noopener">zelenskaya.github.io</a><a href="https://zelenskaya.github.io/experiments/" rel="nofollow noopener">/experiments/</a></p>
<p>If you want to publish your static website in <a href="https://zelenskaya.github.io/experiments/" rel="nofollow noopener">zelenskaya.github.io</a> :</p>
<ol>
<li>
<p>Name your repository <strong><a href="http://zelenskaya.github.io" rel="nofollow noopener">zelenskaya.github.io</a></strong>  </p>
</li>
<li>
<p>In your settings enable GitHub Pages, choose the master branch in the source option</p>
</li>
<li>
<p>When you’ll return to the GitHub pages settings you have to see a green notify about your site</p>
</li>
</ol>

**Comment by zelenskaya** [comment]
<blockquote>
<p><a class="mention" href="/carlogilmar">@carlogilmar</a> wrote:</p>
</blockquote>
<blockquote>
<p>I know that if you enable GitHub Pages over your repository, your url will be: your_github_username.github.io/your_repository_name</p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>You can see this in your case: <a href="https://zelenskaya.github.io/experiments/" rel="nofollow noopener">https://</a><a href="https://zelenskaya.github.io/experiments/" rel="nofollow noopener">zelenskaya.github.io</a><a href="https://zelenskaya.github.io/experiments/" rel="nofollow noopener">/experiments/</a></p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>If you want to publish your static website in <a href="https://zelenskaya.github.io/experiments/" rel="nofollow noopener">zelenskaya.github.io</a> :</p>
</blockquote>
<blockquote>
<ol>
<li>Name your repository <strong><a href="http://zelenskaya.github.io" rel="nofollow noopener">zelenskaya.github.io</a></strong>  </li>
</ol>
</blockquote>
<blockquote>
<ol start="2">
<li>In your settings enable GitHub Pages, choose the master branch in the source option</li>
</ol>
</blockquote>
<blockquote>
<ol start="3">
<li>When you’ll return to the GitHub pages settings you have to see a green notify about your site</li>
</ol>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p> </p>
</blockquote>
<p>Thank you! It helps!</p>
<p>For some reason CSS file is not getting linked, although all works well on my local copy. </p>
<p>Do you have any thoughts?</p>
<p>I understand that this question may go beyond this forum. SO thanks anyway!</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/08e50ea0332716c5ec0ae7ccc530203f7d54a40b.png" data-download-href="/uploads/short-url/1gGzukfH0d4GF2AIjKUQ7a6voAb.png?dl=1" title="Screenshot at Jul 01 22-21-13.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52379364/181066352-a76d5d5c-fc95-4bdc-b4a2-6a3b761cdbf6.png" alt="Screenshot at Jul 01 22-21-13.png" data-base62-sha1="1gGzukfH0d4GF2AIjKUQ7a6voAb" width="690" height="366" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/08e50ea0332716c5ec0ae7ccc530203f7d54a40b_2_690x366.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/08e50ea0332716c5ec0ae7ccc530203f7d54a40b_2_1035x549.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/0/08e50ea0332716c5ec0ae7ccc530203f7d54a40b_2_1380x732.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/08e50ea0332716c5ec0ae7ccc530203f7d54a40b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot at Jul 01 22-21-13.png</span><span class="informations">3358×1786 308 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
